### PR TITLE
⚡️Let's Revive flashbots/pm 🪦➡☀️

### PR DESCRIPTION
**Pull request by dmarzzz**
Let's work more in the open! To accomplish that goal, this PR introduces:
- Community Calls : A new way for Flashbots to coordinate with the public with calls open to anyone
- Repo Clean Up: Fixes a bunch of broken links and gets rid of some unused folders

### CODE REVIEW

File: working-groups/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## Active Working Groups
+
+| Working Group Name | Organizer       | Meeting Cadence    | Milestones                     |
+|--------------------|-----------------|--------------------|--------------------------------|
+| Short Term Censorship Resistance | [Kydo](https://x.com/0xkydo), [Quintus](https://x.com/0xQuintus), [Dmarz](https://x.com/DistributedMarz)        | Weekly, Thursdays  | Initial setup, Beta release    |
+| Encrypted Mempool Standards                | Jane Smith      | Bi-weekly, Mondays | First implementation, Launch   |
+| Flashwares                | Alex Johnson    | Monthly, Fridays   | Draft protocol, Public review  |
```
Comment (Ruteri) [inline working-groups/README.md:13]: We will get into a naming disagreement!

Comment (dmarzzz) [reply working-groups/README.md:13]: about the name of the working group?

File: README.md [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
-* **Coordination**: an organizational effort to streamline operational processes, interface with external teams in the ecosystem and scale ourselves to maximize our impact.
+
+## Our Dual Engines
+Product & research are the semi-autonomous dual engines that propel our organization forward:
+- [Flashbots Product](https://docs.flashbots.net/) builds core infrastructure and ecosystem tooling that redefine the block production supply chain across blockchains.
+- [Flashbots Research](https://github.com/flashbots/mev-research) explores MEV market dynamics and blockchain design challenges at the intersection of economics, security, and cryptography.
 
 Flashbots Research and Product are tightly-coupled dual engines that propel our organization forward in a phased approach:
```
Comment (andytudhope) [inline README.md:18]: Needlessly repetitious imo. One sentence about dual engines is enough :sweat_smile:

File: commnuity-calls/template/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Template
+
+The purpose of the template community call is to provide a template for running your own community call.
+
+## Meetings
+
+| No. |Date | Host | Topic | Agenda | Notes | Recording |
+|:---|:---|:---|---|---|---|:---|
+1 | Today | Liebniz | Example Topic | [agenda](https://example.com) | [write up](https://example.com) | [video](https://example.com) |
```
Comment (andytudhope) [inline commnuity-calls/template/README.md:9]: Happy to take lead here.

[Calendar link](https://calendar.google.com/calendar/render?cid=c_7a8435203986a3f022dcfb47c17f3e238e3aa0659e1523b22ee668193d2ad880@group.calendar.google.com)

We can do the first one after EthCC.

I propose the topic is "MEV SoK". I basically want to look back at different ways of describing MEV from Phil and Xyn, add some more of my own spice, look at things like Peter's tweet, and open up a discussion that will help community members understand the breadth and depth of the design space SUAVE is trying to navigate before we dive into SUAVE-specific stuff in future calls.

File: README.md [hunk 4]
```diff
@@ -17,26 +17,26 @@ Flashbots Research and Product are tightly-coupled dual engines that propel our
 
 Our work is open source and you can follow our progress in each respective repository of the Flashbots organization. Anyone is welcome to comment on an issue or pick it up entirely.
 
-### Community
-The best way to interact with the Flashbots community is through our [forum](https://collective.flashbots.net/), which is our central communication platform for conversations, announcements and information sharing. We also publish monthly [transparency reports](https://collective.flashbots.net/search?q=transparancy%20report%20%23ship%20in%3Atitle) that aggregates information and provides summaries on everything Flashbots and MEV.
+## Community
+The best way to interact with the Flashbots community is through our [forum](https://collective.flashbots.net/), which is our central communication platform for conversations, announcements and information sharing. Additionally we hold recurring [community calls](./commnuity-calls/README.md) on a variety of topics!
```
Comment (andytudhope) [inline README.md:21]: community is spelled wrong in the link

File: README.md [hunk 5]
```diff
@@ -17,26 +17,26 @@ Flashbots Research and Product are tightly-coupled dual engines that propel our
...
+## Useful links
 * 🌐 Website: https://flashbots.net
 * 📄 Docs: https://docs.flashbots.net
+* ☀️ SUAVE Docs: https://suave-alpha.flashbots.net/
 * 📡 Status: https://status.flashbots.net
 * 💬 Discord: https://discord.gg/flashbots
 * 💡 Forum: https://collective.flashbots.net
 * 📝 Writings: https://writings.flashbots.net
 * ⚡ MEV-Boost: https://boost.flashbots.net
-* 🗓️ Calendar: https://calendar.flashbots.net
```
Comment (andytudhope) [inline README.md:37]: Why remove the calendar? Seems useful to me

File: commnuity-calls/README.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+| [Flashwares](./flashwares/README.md)         | [SUAVE Andromeda]()     |  TEE Education and Design Discussions  | Burst Installments   | [Forum](https://collective.flashbots.net/tag/flashwares) |
+| [SUAVE.dev](./suapp-developer-streams/README.md)         | [cryptowanderer](https://x.com/cryptowanderer) & [Brock](https://x.com/zeroXbrock/)     | SUAPP Development Office Hours and Tutorials   |  Monthly  | [Community Calender](http://calendar.flashbots.net/) |
+| [All Core Devs](./All-Core-Devs/README.md)         | [dmarz](https://x.com/DistributedMarz) | Core Protocol Development Discussions   |  Monthly  | [Community Calender](http://calendar.flashbots.net/) |
+
+## 🧊 Retired Community Calls
+
+| Name | Organizer       | Purpose                     |
+|--------------------|-----------------|--------------------------------|
+| [MEV Roasts](./mev-roasts/README.md)         | [Flashbots](https://writings.flashbots.net/)     |  Communication bridge between Flashbots and the broader community   |
+
```
Comment (andytudhope) [inline commnuity-calls/README.md:18]: Could be worth listing the abandoned whiteboarding sessions if only for inspo? Only one recording was made public though: https://collective.flashbots.net/t/whiteboard-an-overview-of-credible-commitment-devices/926

File: README.md (hunk 5 above)
Comment (dmarzzz) [reply README.md:37]: oh whoops didnt try this

File: README.md (hunk 4 above)
Comment (dmarzzz) [reply README.md:21]: ty

File: commnuity-calls/README.md (hunk 6 above)
Comment (dmarzzz) [reply commnuity-calls/README.md:18]: if only 1 i'd rather not

Comment (andytudhope) [reply commnuity-calls/README.md:18]: The only reason I suggest it is that it demonstrates that there are a variety of formats which could work well for community calls. No strong opinion though

**Review by sukoneck — APPROVED** [review]

**Comment by freddmannen** [comment]
https://transparency.flashbots.net/ should be removed from the list of links as it's deprecated

**Review by metachris — APPROVED** [review]

**Comment by dmarzzz** [comment]
closing to take some time to take a step back and understand if this is the best approach

**Pull request closed without merging**